src/fu-util.c

```diff
@@ -2778,6 +2780,9 @@ main (int argc, char *argv[])
 		{ "disable-ssl-strict", '\0', 0, G_OPTION_ARG_NONE, &priv->disable_ssl_strict,
 			/* TRANSLATORS: command line option */
 			_("Ignore SSL strict checks when downloading files"), NULL },
+		{ "ipfs", '\0', 0, G_OPTION_ARG_NONE, &enable_ipfs,
+			/* TRANSLATORS: command line option */
+			_("Use IPFS when downloading files"), NULL },
```

Comment:
Is the average fwupdmgr user going to know what IPFS is?

Reply:
It's one short google away -- and I don't think that they'll know what "Ignore SSL strict checks" would do either without some research.